tests/CommentsGridFieldActionTest.php

```diff
@@ -40,6 +41,9 @@ class CommentsGridFieldActionTest extends SapphireTest
     protected function setUp(): void
     {
         parent::setUp();
+        if (Deprecation::isEnabled()) {
+            $this->markTestSkipped('Test calls deprecated code');
+        }
         $this->list = new DataList(Team::class);
```

Comment:
Could we also mark as deprecated `$list` property https://github.com/silverstripe/silverstripe-comments/blob/b771bb624afc8c8deaa368883a6044e331f1a2e0/tests/CommentsGridFieldActionTest.php#L33

Reply:
It's a unit test, no need to deprecate things